src/lib/autocomplete/autocomplete-trigger.ts

```diff
@@ -231,7 +231,9 @@ export class MdAutocompleteTrigger implements ControlValueAccessor, OnDestroy {
   }
 
   _handleKeydown(event: KeyboardEvent): void {
-    if (this.activeOption && event.keyCode === ENTER) {
+    if (event.keyCode === ESCAPE) {
+      this.panelOpen ? this.closePanel() : this._element.nativeElement.blur();
```

Comment:
I tested that manually, and it seems to go to the next in the tab order on TAB. So I think it's okay.